README.md

```diff
@@ -1,3 +1,8 @@
+# Plugin Website
+
+A gallery of current plugins can be found here: https://plugins.manim.community/
```

Comment:
I think it should be at the bottom of the readme.

Reply:
I tend to agree with @naveen521kk if you insist on having its own `# Plugin Website` heading, or better yet, it could be an admonition and placed here:

> Plugins are features that extend Manim's core functionality. This is a template project that demonstrates how you can create and upload a manim plugin to PyPI using a PEP 517 compliant build system, Poetry.
>
>Poetry is NOT required to create plugins, but is recommended because it provides build isolation and ensures users can reliably install your plugin without falling into dependency hell. You may use another build system (e.g. Flit, Setuptools, Pipenv, etc...) if you wish.
>
> --> HERE IS A GOOD SPOT

Reply:
Ok, I've moved it.
I think it deserves its own heading, as this is a good very good reference for people who consider making a plugin.